lib/session/call-info.js

```diff
@@ -144,7 +144,7 @@ class CallInfo {
     if (typeof this.duration === 'number') obj.duration = this.duration;
 
     if (this._customerData) {
-      Object.assign(obj, {customerData: this._customerData});
+      Object.assign(obj, {customerdata: this._customerData});
```

Comment:
I don't understand this.  There are many places in the code where we are referencing 'customerData', none where we are referencing 'customerdata'.  Example, in call-info.js
```
      if (tag) this._customerData = tag;
```
Its unclear to me why this changes fixes the bug, and why it may not be creating other bugs given that we are now referring to the collection of customer tagged data in two different ways in the code